go/ql/lib/semmle/go/dataflow/internal/FlowSummaryImplSpecific.qll

```diff
@@ -84,13 +58,14 @@ DataFlowType getCallbackReturnType(DataFlowType t, ReturnKind rk) { none() }
 
 /**
  * Holds if an external flow summary exists for `c` with input specification
- * `input`, output specification `output`, and kind `kind`.
+ * `input`, output specification `output`, kind `kind`, and a flag `generated`
+ * stating whether the summary is autogenerated.
  */
-predicate summaryElement(DataFlowCallable c, string input, string output, string kind) {
+predicate summaryElement(Callable c, string input, string output, string kind, boolean generated) {
```

Comment:
```suggestion
predicate summaryElement(SummarizedCallableBase c, string input, string output, string kind, boolean generated) {
```
Probably better to implement the interface in terms of the types the library expects